tests/bsim/bluetooth/mesh/src/test_dfu.c

```diff
@@ -1034,6 +1050,83 @@ static void test_cli_fail_on_persistency(void)
 	PASS();
 }
 
+static void test_cli_all_targets_lost(void)
```

Comment:
Is it possible to split this test into smaller functions by analogy with `test_target_fail_on_<smth>`? It is probably matter of taste, but I don't really see benefit of this giant `switch` and the new option.